server/apolloServer.js

```diff
@@ -30,6 +33,15 @@ function newApolloServer (app, schema, httpServer, tracing, playgroundConfig, gr
       if (AuthContextProvider) {
         context.auth = new AuthContextProvider(req)
       }
+      if (auditLogEnabled) {
```

Comment:
I personally preffer this to happen in the log module (so all logic will be there)
Then this code could look like this:
```
 context.clientInfo = req.clientInfo
context.auditLog = auditLog // Required from log (it can be NOP etc.)
```

Then this if is not executed for every request and we enclose all logging logic in this component. We do not have component based server but we could write stuff this way so if number of logic will grow we can extract that to separate elements easily.

Reply:
+1 will do